blackbox/docs/admin/monitoring.rst

```diff
@@ -272,6 +272,62 @@ of each thread pool with the following attributes:
++======================+==========================================================+
+| ``Parent``           | Statistics of the ``parent`` circuit breaker             |
+|                      | containing summarized counters accross all circuit       |
+|                      | breakers.                                                |
++----------------------+----------------------------------------------------------+
+| ``Query``            | Statistics of the ``query`` circuit breaker used to      |
+|                      | account memory usage of SQL execution including          |
+|                      | intermediate states e.g. on aggreation and resulting     |
+|                      | rows.                                                    |
++----------------------+----------------------------------------------------------+
+| ``JobsLog``          | Statistics of the ``jobs_log`` circuit breaker used to   |
+|                      | account memory usage of the ``sys.jobs_log`` table.      |
++----------------------+----------------------------------------------------------+
+| ``OperationsLog``    | Statistics of the ``operations_log`` circuit breaker     |
+|                      | used to account memory usage of the                      |
+|                      | ``sys.operations_log`` table.                            |
++----------------------+----------------------------------------------------------+
+| ``FieldData``        | Statistics of the ``field_data`` circuit breaker used    |
+|                      | for estimating the amount of memory a field will require |
+|                      | to be loaded into memory.                                |
++----------------------+----------------------------------------------------------+
+| ``InFlightRequests`` | Statistics of the ``in_flight_request`` circuit breaker  |
+|                      | used to account memory usage of all incoming requests    |
+|                      | on transport or HTTP level.                              |
++----------------------+----------------------------------------------------------+
+| ``Requests``         | Statistics of the ``request`` circuit breaker used to    |
+|                      | account memory usage of per-request data strucutre.      |
++----------------------+----------------------------------------------------------+
+
+Each of them returns a `CompositeData`_ object containing detailed statistics
+of each circuit breaker with the following attributes:
+
++------------------+------------------------------------------------------+
+| Name             | Description                                          |
++==================+======================================================+
+| ``name``         | The circuit breaker name this statistic belongs to.  |
++------------------+------------------------------------------------------+
+| ``estimated``    | The currently accounted memory estimations.          |
```

Comment:
this is a bit confusing as the name doesn't imply it's the estimated "used" memory. Would it make sense to rename it to ``used``?

Reply:
wasn't sure about that, can change it if you prefer

Reply:
I think `used` makes more sense (@joemoe what do you think?)